content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -54,25 +173,23 @@ Long term, we may create more specialised customer success roles to drive high l
 - webinars and 1-to-many sessions
 - working with product growth to build self-serve ways to get to value, and address common issues
 
+We also expect to automate most of the manual steps in the signup workflow to take CEs out of that loop.
+
 ### Customer Support (CS)
 
 - Cloud trial users that ecounter issues can use our discord, or send an email to support@sourcegraph.com. The support team is expected to address those issues within normal [SLAs](../../../ce-support/support/index.md#sts=SLAs).
-- If a user tries to join an existing instance and is blocked, they will request help on support@sourcegraph.com. The support team should reach out to the user and work with #growth to make sure the instance admin knows that user has requested an invite and is blocked. This will be automated.
+- If a user tries to join an existing instance and is blocked, they might request help on support@sourcegraph.com. The support team should reach out to the user and work with #growth to make sure the instance admin knows that user has requested an invite and is blocked. This will be automated.
```

Comment:
This is automated now or will be automated when this launches?

Reply:
All users are redirected to the instance. However, if SSO is not setup they can't join. The AE/CE though can help here.
We'll make it self-serve soon: https://github.com/sourcegraph/sourcegraph/issues/42981

Do you see a way to clarify this in the next? Fee free to directly add it in if you'd like